packages/oft-evm/artifacts/contracts/OFT.sol/OFT.dbg.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I think these dbg files should be ignored, the rest of the artifacts is fine to keep BUT what we need to do to stay consistent & compatible with devtools is to move `artifacts/contracts` to just `artifacts` and export that folder in `package.json`.

There are two ways I can think of:

- Update `hardhat.config.ts` and change the artifacts folder in the `paths` config to something else (let's say `fartifacts`), then on build copy `fartifacts/contracts` to `artifacts`
- Get rid of hardhat completely and do the same copying but form the `out` folder of foundry

The copying can be done either by low-level (but system-specific) `rm -rf && cp -R` or then by adding a build step and a tsup config, both have pros and cons

Reply:
Not sure I follow.  What about just eliminating hardhat, then changing foundry.toml so `out=artifacts`?

Reply:
Yeah we can do that too. The trick for that solution is not to pack all the artifacts with the package since foundry will include all the test contracts there too:

<img width="167" alt="Screenshot 2024-07-10 at 1 51 14 PM" src="https://github.com/LayerZero-Labs/devtools/assets/1451480/0b03b259-be55-4785-b46c-643dc7463b83">

So you need to handpick the artifacts you want included in `package.json` (in this case it should be just the `OFT` though)

Reply:
done.